charset_normalizer/cd.py

```diff
@@ -62,7 +62,7 @@ def unicode_range_languages(primary_range: str) -> List[str]:
     return languages
 
 
-@lru_cache()
+@lru_cache(maxsize=4096) # up to 2999 IANA char sets
```

Comment:
I did not limit the cache size because I did not want to set an arbitrary size.
The cPython ship with around ~100 text codecs, and the number vary depending of what gets deprecated.

The rule I fixed myself regarding the cache size is: 

- If there is a chance that it would get out of control (ram usage) then limit it for sure
- Or if the max expected size is constant, eg. utf-8 max allocation is known in advance.

My opinion here is to let the size undefined, I don't expect more than 100 entries. WDYT?

Reply:
OK, I'll leave the LRU cache statement as it was.